### PR TITLE
feat(ghb): Grindavik Housing Buyout new application

### PR DESCRIPTION
**Pull request by Toti91**
# Grindavik Housing Buyout application

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by baering — APPROVED** [review]
Approved for changes in two core files ✅

**Review by Thorrik58 — APPROVED** [review]

File: libs/application/template-api-modules/src/lib/modules/templates/grindavik-housing-buyout/grindavik-housing-buyout.service.ts [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+      auth.nationalId === '0101302399'
+    ) {
+      return {
+        realEstateId: 'F12345',
+        realEstateAddress: 'Vesturhóp 34, 240 Grindavík',
+      }
+    }
+
+    const dateInQuestion = '2023-10-11'
+    const domicileOn10nov = getDomicileOnDate(data, dateInQuestion)
```
Comment (baering) [inline libs/application/template-api-modules/src/lib/modules/templates/grindavik-housing-buyout/grindavik-housing-buyout.service.ts:90]: The date you are checking is October 11 but the variable name suggests you want to check november 10.

Comment (obmagnusson) [reply libs/application/template-api-modules/src/lib/modules/templates/grindavik-housing-buyout/grindavik-housing-buyout.service.ts:90]: Good catch

File: libs/application/template-api-modules/src/lib/modules/templates/grindavik-housing-buyout/grindavik-housing-buyout.service.ts [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+              ),
+          },
+          title: notEligible.noResidenceTitle,
+        },
+        400,
+      )
+    }
+
+    return {
+      realEstateId: domicileOn10nov?.realEstateNumber ?? '12345',
```
Comment (baering) [inline libs/application/template-api-modules/src/lib/modules/templates/grindavik-housing-buyout/grindavik-housing-buyout.service.ts:110]: Are you supposed to get through this check without any residence?

Comment (obmagnusson) [reply libs/application/template-api-modules/src/lib/modules/templates/grindavik-housing-buyout/grindavik-housing-buyout.service.ts:110]: We are just getting it on main so texts and the flow can be reviewed.

**Pull request merged**